File: fix_css.php
<?php
$filePath = 'c:/xampp/htdocs/EducationalMinisterialOfficersUttarakhand/public/css/public.css';
$lines = file($filePath);
if ($lines === false) {
    die("Could not read file.");
}

// We want to keep up to line 838 (index 837)
// and handle line 839 (index 838) if it has the closing brace
$goodLines = array_slice($lines, 0, 839);

// Check if the last line is corrupted
$lastLine = $goodLines[count($goodLines) - 1];
if (strpos($lastLine, '@ m e d i a') !== false) {
    // Truncate to 838 lines
    $goodLines = array_slice($lines, 0, 838);
    // Ensure it ends with a closing brace
    $lastGoodLine = trim($goodLines[count($goodLines) - 1]);
    if ($lastGoodLine !== '}') {
        $goodLines[] = "\n}\n";
    }
}

if (file_put_contents($filePath, implode("", $goodLines))) {
    echo "File cleaned up successfully.\n";
} else {
    echo "Failed to write to file.\n";
}

File: fix_css.py
import os

file_path = r'c:\xampp\htdocs\EducationalMinisterialOfficersUttarakhand\public\css\public.css'
with open(file_path, 'r', encoding='utf-8', errors='ignore') as f:
    lines = f.readlines()

# We want to keep up to line 839 (index 838) but only the first character '}'
# Or better, just keep lines 0 to 838 (the last good brace)
good_lines = lines[:839]
# Check if the last line is corrupted
if '@ m e d i a' in good_lines[-1]:
    # Truncate to 838 lines
    good_lines = lines[:838]
    # Ensure it ends with a closing brace
    if not good_lines[-1].strip() == '}':
        good_lines.append('\n}\n')

with open(file_path, 'w', encoding='utf-8') as f:
    f.writelines(good_lines)

print("File cleaned up successfully.")
